header.php

```diff
@@ -57,5 +57,51 @@
 
   <!-- start content -->
   <header id="header">
+  <?php
+    $logo = IGV_get_option('_igv_metadata_logo');
```

Comment:
def do this cus otherwise the logo image could be all over the place and also will include src-set support 👍